doc/random.md

```diff
@@ -65,7 +65,7 @@ Returns an unsigned 32 bit integer random number.
 <a name="torch.uniform"></a>
 ### [number] uniform([a],[b]) ###
 
-Returns a random real number according to uniform distribution on [a,b[. By default `a` is 0 and `b` is 1.
+Returns a random real number according to uniform distribution on [a,b]. By default `a` is 0 and `b` is 1.
```

Comment:
Hi, the original `[a,b[` here is correct as @fmassa pointed out in https://github.com/torch/torch7/pull/427. [The code](https://github.com/torch/torch7/blob/master/lib/TH/THRandom.c#L203) shows it's an open interval in `b`.

Is `[a,b)` clearer than `[a,b[` here? I haven't seen the reversed-bracket notation for open intervals before, and it seems like this is confusing other people as well.